CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.3] - 2024-04-16
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.5.2...v1.5.3)
+
+### Added
+
+* Apply tags to requests that have been intentionally throttled (https://github.com/openfga/openfga/pull/1531). This will add a new field "throttled" to such logs.
+
+### Fixed
+
+* Fix panic that occurred on Check API with some authorization models and tuples (https://github.com/openfga/openfga/pull/1517)
+
+### Changed
+
+* Remove experimentals for modular models/schema 1.2 (https://github.com/openfga/openfga/pull/1520)
```

Comment:
Can we clarify that it is on by default?


"Modular Models (Schema 1.2) support is enabled by default and the experimental flag for it has been dropped"